### PR TITLE
foreman-discovery: release 2.0.0.rc1

### PR DESCRIPTION
**Pull request by GregSutcliffe**
No assets for easy release \o/

### CODE REVIEW

File: plugins/ruby-foreman-discovery/debian/changelog [hunk 1]
```diff
@@ -1,3 +1,9 @@
+ruby-foreman-discovery (2.0.0~rc1-1) stable; urgency=low
+
+  * Set HOME to ~foreman as rb-readline requires it
```
Comment (domcleal) [inline plugins/ruby-foreman-discovery/debian/changelog]: update this

**Comment by domcleal** [comment]
postinst needs db:migrate, db:seed and apipie:cache rake tasks adding.

File: plugins/ruby-foreman-discovery/debian/changelog (hunk 1 above)
Comment (GregSutcliffe) [inline plugins/ruby-foreman-discovery/debian/changelog]: Whoops, thanks

**Comment by domcleal** [comment]
Is this also for deb/1.7, like the RPMs? (https://github.com/theforeman/foreman-packaging/pull/288#issuecomment-70805889)

**Comment by GregSutcliffe** [comment]
Indeed. Technically according to the engine.rb it's for 1.6 too, but no one has checked if it works ;)

**Comment by domcleal** [comment]
Thanks @GregSutcliffe, merged also to deb/1.7 as 4b8bef8.  I'll build it now.

**Pull request merged**